recipes/apsg/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "apsg" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d2c0b8f68fa3dc68e44d6ee83e5b01d894b1f2567f89fc97ff00749528dcef06
+
+build:
+  number: 0
+  noarch: python
```

Comment:
I'm not certain if noarch supports `entry_points` currently, don't remove it just yet though. Someone wiser might know.

Reply:
It does

Reply:
It is not really CLI app. CLI is used just to open an interactive shell and import APSG. I'm not sure how to test it.